packages/components/src/IconList/tests/jest/IconList.ssr.test.tsx

```diff
@@ -0,0 +1,20 @@
+/**
+ * @jest-environment node
+ */
+import { IconList } from "../../src/IconList.tsx";
+import { renderToString } from "react-dom/server";
+import { AddIcon, PlusIcon } from "@hopper-ui/icons";
+
+describe("IconList", () => {
+    it("should render on the server", () => {
+        const renderOnServer = () =>
+            renderToString(
+                <IconList>
+                    <AddIcon />
+                    <PlusIcon />
+                </IconList>
+            );
+
+        expect(renderOnServer).not.toThrow();
```

Comment:
I never did understand the purpose of these ssr tests.

Reply:
SSR catch small mistakes that could happen. For instance, calling "useEffect" when using SSR results in a warning being throw. This ensure things such as "windows" or "document" aren't called directly in the component.